appium/tests/specs/inbox/ReadAttachmentEmail.spec.ts

```diff
@@ -0,0 +1,62 @@
+import {
+    SplashScreen,
+    CreateKeyScreen,
+    InboxScreen,
+    EmailScreen,
+    AttachmentScreen
+} from '../../screenobjects/all-screens';
+
+import {CommonData} from '../../data';
+
+describe('INBOX: ', () => {
+
+    it('user is able to view encrypted email with attachment', () => {
+
+        const senderEmail = CommonData.sender.email;
+        const emailSubject = CommonData.encryptedEmailWithAttachment.subject;
+        const emailText = CommonData.encryptedEmailWithAttachment.message;
+        const attachmentName = CommonData.encryptedEmailWithAttachment.attachmentName;
+
+        const wrongPassPhrase = CommonData.account.wrongPassPhrase;
+        const correctPassPhrase = CommonData.account.passPhrase;
+        const bundleId = CommonData.bundleId.id;
+
+        SplashScreen.login();
+        CreateKeyScreen.setPassPhrase();
```

Comment:
That one is `ImportKeyFromEkmSetupScreen`. Because we also have a create key flow that we're not testing yet, and it's different.